app/registerServiceWorker.js

```diff
@@ -28,18 +28,10 @@ function registerValidSW(swUrl) {
         const installingWorker = registration.installing;
         installingWorker.onstatechange = () => {
           if (installingWorker.state === 'installed') {
-            if (navigator.serviceWorker.controller) {
-              // At this point, the old content will have been purged and
-              // the fresh content will have been added to the cache.
-              // It's the perfect time to display a "New content is
-              // available; please refresh." message in your web app.
-              console.log('New content is available; please refresh.');
-            } else {
-              // At this point, everything has been precached.
-              // It's the perfect time to display a
-              // "Content is cached for offline use." message.
-              console.log('Content is cached for offline use.');
-            }
+            // At this point, everything has been precached.
```

Comment:
most of this code is just left here to unregister all "outdated" clients with the serviceWorkers?

and the section that is removed is not actually being used by anything right?

Reply:
Yes that's correct. After this is released then I can remove this file entirely in a future update